SportsHub/SportsHub.Api/Controllers/UserController.cs

```diff
@@ -10,6 +11,28 @@ namespace SportsHub.Api.Controllers
     [ApiController]
     public class UserController : ControllerBase
     {
+        private readonly IUnitOfWork unitOfWork;
+
+        public UserController(IUnitOfWork unitOfWork)
+        {
+            this.unitOfWork = unitOfWork;
+        }
+
+        [HttpGet("GetUserByPassword")]
+        public IActionResult GetUserByPassword(string password)
+        {
```

Comment:
move to service

Reply:
also, we missed UserId or UserName here) there no sense getByPassword because few users can have the same password

Reply:
Implemented